txnsync/emulatorTimer_test.go

```diff
@@ -112,6 +112,7 @@ func (g *guidedClock) Advance(adv time.Duration) {
 	// fire expired clocks
 	for _, entry := range expiredClocks {
 		entry.ch <- g.zero.Add(g.adv)
+		close(entry.ch)
```

Comment:
closing the channel is important, since it's handling the case where we have two "users" for the same timeout value -> in which case we need the channel to be closed.